lib/kv-extreme-scale.js

```diff
@@ -2,9 +2,12 @@
 // Node module: loopback-connector-kv-extreme-scale
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
-
 'use strict';
 
+var SG = require('strong-globalize');
+SG.SetRootDir(__dirname + '/..');
+var g = SG();
```

Comment:
Thanks for fixing it via https://github.com/strongloop/strong-globalize/pull/92, I remember it was broken a long time ago so I didn't bother moving it out.